engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql

```diff
@@ -18,4 +18,10 @@
 --;
 -- Schema upgrade from 4.15.0.0 to 4.15.1.0
 --;
+-- Add support for VMware 7.0
```

Comment:
@nvazquez any new OS supported with 7.0 / 7.0.1.0 to be added to guest os mappings

Reply:
@sureshanaparti yes - added on the last commit